README.md

```diff
@@ -42,15 +41,57 @@ $ WS_SECRET="secret2" npm start
 ## Flags
 
 - db-endpoint: Database endpoint to store the data.
-
 - collector.addr (default=localhost:8000): Websocket address to collect metrics.
-
 - collector.secret (default=''): Secret for the local websocket collector.
-
 - log-level (default=info): Level to log the output.
-
 - frontend.addr: Address of the ethstats frontend to proxy the data.
-
 - frontend.secret: Secret to be used in the ethstats proxy.
-
 - save-block-txs: Whether block transactions should be written to database.
+
+## Architecture
+
+![](docs/architecture.svg)
+
+<details>
+<summary><code>source</code></summary>
+
+```plantuml
+@startuml docs/architecture
+' Generate this file with: platnuml -tsvg README.md
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Component.puml
+
+title Ethstats Architecture
+
+LAYOUT_WITH_LEGEND()
+
+System_Boundary(blockchain, "Polygon PoS v1") {
+  Container_Ext(node1, "Full Node 1")
+  Container_Ext(node2, "Full Node 2")
+  Container_Ext(node3, "Full Node 3")
+}
+
+System_Boundary(ethstats, "Ethstats") {
+  Container(backend, "Ethstats Backend")
+  Container(hasura, "Hasura")
+  Container(postgres, "Postgres")
+  Container(frontend, "Ethstats Frontend")
```

Comment:
What is this specifically? If it's https://bor-mainnet.vitwit.com/ it might consider drawing this outside of the "ethstats" system more like the Datadog box actually.

Reply:
Originally, I thought we were deploying the `ethstats-frontend` because of the `docker-compose` but after further inspection of the `taskdef.json` and `deployment.yaml`, I realized we're only creating the `ethstats-backend` and `graphql-engine` containers.